tests/test_cluster_utils.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import pytest
+
+from ocsci.testlib import tier1, ManageTest
+from ocs.cluster import CephCluster
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Create disks
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    global cluster
+    cluster = CephCluster()
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    pass
+
+
+@tier1
+@pytest.mark.usefixtures(
+    test_fixture.__name__,
+)
+class TestClusterUtils(ManageTest):
+
+    username = "client.test"
+    caps = "mon 'allow r' osd 'allow rwx'"
+
+    def test_user_creation(self):
+        assert cluster.create_user(self.username, self.caps)
+
+    def test_get_user_key(self):
+        key = cluster.get_user_key(self.username)
```

Comment:
This is not good approach to count that this user is created by some of your previous test. If you count on some user here this should be done in setup fixture which should provide you the user.  Can be class level fixture if used in multiple tests.

Reply:
Done, handled it in setup and teardown